.github/workflows/maven.yml

```diff
@@ -15,24 +15,24 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: 'szenius/set-timezone@v1.2'
-      with:
-        timezoneLinux: Europe/Zurich
-        timezoneMacos: Europe/Zurich
-        timezoneWindows: W. Europe Standard Time
-    - uses: actions/checkout@v4
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v4
-      with:
-        java-version: 8
-        distribution: 'zulu'
-        java-package: jdk
-        architecture: x64
-    - name: Build with Maven
-      run: mvn -B package --file pom.xml
-    - name: Upload successfully built JAR
-      uses: actions/upload-artifact@main
-      with:
-        name: xlconnect_jar
-        path: | 
-          /home/runner/work/xlconnect-java/xlconnect-java/target/XLConnect-*.jar
+      - uses: 'szenius/set-timezone@v1.2'
+        with:
+          timezoneLinux: Europe/Zurich
+          timezoneMacos: Europe/Zurich
+          timezoneWindows: W. Europe Standard Time
+      - uses: actions/checkout@v4
+      - name: Set up JDK 17
```

Comment:
Note that we are setting the `maven.compiler.release`  property in `pom.xml`  to make sure the resulting JAR is Java 8 compatible.